README.md

```diff
@@ -63,14 +202,30 @@ module "eks" {
     }
   }
 
-  # Cluster access entry
-  # To add the current caller identity as an administrator
-  enable_cluster_creator_admin_permissions = true
+  tags = {
+    Environment = "dev"
+    Terraform   = "true"
+  }
+}
+```
+
+### Cluster Access Entry
+
+When enabling `authentication_mode = "API_AND_CONFIG_MAP"`, EKS will automatically create an access entry for the IAM role(s) used by managed node group(s) and Fargate profile(s). There are no additional actions required by users. For self-managed node groups and the Karpenter sub-module, this project automatically adds the access entry on behalf of users so there are no additional actions required by users.
+
+On clusters that were created prior to CAM support, there will be an existing access entry for the cluster creator. This was previously not visible when using `aws-auth` ConfigMap, but will become visible when access entry is enabled.
```

Comment:
What is CAM support?

Reply:
CAM = cluster access management. I'll update to use the expanded form for clarity

Reply:
Cluster Auto Mode, i assume

Reply:
corrected in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/3225/commits/c24470be1e0a8204beb7f6a6d15cd039ae44f5d8